terraform/environments/production/dns.tf

```diff
@@ -190,6 +190,31 @@ resource "google_dns_record_set" "sendgrid-reverse-dns" {
   ttl     = 3600
 }
 
+# Mailgun
+
+resource "google_dns_record_set" "mailgun-dkim" {
+  project      = module.google-cloud-project.project.project_id
+  managed_zone = module.google-cloud-dns.zone_name
+
+  name = "kone._domainkey.${module.google-cloud-dns.dns_name}"
+  type = "TXT"
+  ttl  = 3600
+
+  rrdatas = [
+    "k=rsa; p=MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAwhTddUFz+LHTx63SpYvoAc4UzPgXy71Lq950bgVgrwHqLiktRnXFliKGbwL/QPyzOOWBYd1B3brC81B0IoBZkNxFj1mA1EKd8oFi8GMaKA5YuPbrkTT9AGXx0VpMMqDUcYoGWplXnMSY2ICdSRxOdQ5sXLdLqEyIVWm8WiF2+U7Zq15PSNr1VigByCknc7N0Pes0qbVTuWVNdBBYFO5igHpRaHZtYU/dT5ebXxcvZJgQinW23erS6fFgNuUOOwhGJCay5ahpAnufuQB52eEkM/AHb9cXxVG5g04+6xZSMT7/aI7m1IOzulOds71RAn7FN4LJhdI0DgOmIUVj4G32OwIDAQAB"
```

Comment:
This will likely not apply, I recommend running it locally to test if DNS changes will take into effect:
```
terraform apply -var image_tag=$(terraform output -raw image_tag)
```

Reply:
Yep, you were correct, it didn't like the string as it was.  Good catch!  Had to google to figure out why and saw that there was a 255 char max for any string in the record, so I put a comment above the entries explaining why.

I applied the changes to staging, but I figured I would let CI do Prod since there were changes it wanted to make that I wasn't sure of and wanted to confirm with you that they are expected.